analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -120,10 +122,9 @@ abstract class PackageManager(
     }
 
     /**
-     * Resolve dependencies for single [definitionFile], amending the [result].
+     * Resolve dependencies for single [definitionFile], returning the [AnalyzerResult].
```

Comment:
Mind adding "a" along the way? -> "for a single"

Reply:
Done.